lib/Net/servent.t

```diff
@@ -15,8 +18,9 @@ BEGIN {
     unless ($hasse) { print "1..0 # Skip: no netdb.h\n"; exit 0 }
 }
 
+our @servent;
 BEGIN {
```

Comment:
Seems easier to get rid of the whole begin block?

Reply:
This does make sure the (oddly placed) `use Net::servent` does not execute until after this.

Reply:
Right but the module doesn't do anything other than load Exporter (strangely) at BEGIN time.